### PR TITLE
add distance and points to levelup dialog

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

**Comment by varffs** [comment]
there is a function `Utilities.Number.getRandomInt(min, max)` that does that, which you can use in `Game.getRandomDistance()` And the function name getRandomDistance is a really confusing name cus it sounds like getting a random distance not returning a string describing distance travelled

**Comment by mikeyrayvon** [comment]
then what do you want to call it?

**Comment by varffs** [comment]
something like `getTotalDistanceDescription`

File: cordova/www/js/src/game.js [hunk 1]
```diff
@@ -80,21 +80,21 @@ Game = {
...
-    var oldDistance = _this.getDistance();
+    var oldDistance = _this.getTotalDistance();
     var newDistance = parseFloat(newDistance);
 
     window.localStorage.setItem('distance', oldDistance + newDistance);
   },
 
-  getDistance: function() {
+  getTotalDistance: function() {
     return window.localStorage.getItem('distance') ? window.localStorage.getItem('distance') : 0;
```
Comment (varffs) [inline cordova/www/js/src/game.js]: if this is a integer then this should be parseInt otherwise it can cause issues later when maths happens

**Comment by varffs** [comment]
The function occasionally returns `TypeError: undefined is not an object (evaluating 'randomDistance[0]')` which i think is cus the .length should be -1 but not sure if that is it.

**Comment by varffs** [comment]
Interesting to note tho the probably cause for clashes between built files. The autoprefixer configuration is clearly different for use between computers. some do ms, some dont bother etc. Perhaps it needs an explicit config set in the gulpfile to override globals, or we have version mismatches and are using the 'last 2 versions' type rule

**Pull request merged**